tests/options/sortAttributes/sort-attributes.test.ts

```diff
@@ -38,19 +38,62 @@ describe('Options', () => {
 
 	describe('sort utilities', () => {
 		test('compare 1', () => {
+			const pugSortAttributesBeginning = ['v-for', ':key', 'src', 'alt'];
+			const pugSortAttributesEnd: string[] = [];
 			const expected: ReadonlyArray<string> = ['v-for', ':key', 'src', 'alt'];
 			const code: string[] = ['alt', ':key', 'v-for', 'src'];
 			const actual: string[] = code.sort((a, b) =>
-				compareAttributeToken(createAttributeToken(a), createAttributeToken(b), ['v-for', ':key', 'src', 'alt'])
+				compareAttributeToken(
+					createAttributeToken(a),
+					createAttributeToken(b),
+					pugSortAttributesEnd,
+					pugSortAttributesBeginning
+				)
 			);
 
 			expect(actual).toStrictEqual(expected);
 		});
 		test('compare 2', () => {
+			const pugSortAttributesBeginning: string[] = [];
+			const pugSortAttributesEnd = ['v-for', ':key', 'src', 'alt', '@click', ':disabled'];
 			const expected: ReadonlyArray<string> = ['v-for', ':key', 'src', 'alt', '@click', ':disabled'];
 			const code: string[] = ['v-for', ':disabled', ':key', '@click', 'src', 'alt'];
 			const actual: string[] = code.sort((a, b) =>
-				compareAttributeToken(createAttributeToken(a), createAttributeToken(b), ['@click', ':disabled'], true)
+				compareAttributeToken(
+					createAttributeToken(a),
+					createAttributeToken(b),
+					pugSortAttributesEnd,
+					pugSortAttributesBeginning
+				)
+			);
+
+			expect(actual).toStrictEqual(expected);
+		});
+		test('compare 3', () => {
+			const pugSortAttributesBeginning = ['^x$', '^y$', '^z$'];
+			const pugSortAttributesEnd = ['v-for', ':key', 'src', 'alt', '@click', ':disabled'];
+			const expected: ReadonlyArray<string> = [
+				'x',
+				'y',
+				'z',
+				'a',
+				'b',
+				'c',
```

Comment:
Why would `a` sorted before `c` if you didn't configured it in any way in `beginning` or `end`?
I won't expect this `expected`

I would expect this: `'x', 'y', 'z', 'c', 'a', 'b', 'v-for', ':key', 'src', 'alt', '@click', ':disabled'`

Reply:
I was considering that middle attributes should be sorted alphabetically, which I was, indeed, wrong.

_this is what happens when you try to mess with someone's project while sleepy, sorry_